ia/src/protocole.py

```
@@ -115,11 +96,20 @@ def handle_client(client) -> None:
             message, buffer = buffer.split("\n", 1)
 
             if message:
-                if (message == "dead\n"):
+                if (message == "dead"):
```

Pylint (3.3.7)

[convention] Unnecessary parens after 'if' keyword

(C0325)